data/json/hobbies.json

```diff
@@ -90,6 +93,357 @@
+    "points": 5,
+    "skills": [
+      { "level": 2, "name": "melee" },
+      { "level": 3, "name": "bashing" },
+      { "level": 3, "name": "stabbing" },
+      { "level": 3, "name": "cutting" }
+    ],
+    "traits": [ "MARTIAL_ARTS5" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "gourmand",
+    "name": "Gourmand",
+    "description": "You truly enjoy eating and can eat faster, can eat and drink more, than anyone else!",
+    "points": 3,
+    "traits": [ "GOURMAND" ],
+    "skills": [ { "level": 1, "name": "cooking" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "hates_books",
+    "name": "Hates Books",
+    "description": "Reading is for nerds!  Boring books are more boring, and you can't have fun by reading books.",
+    "points": -1,
+    "traits": [ "HATES_BOOKS" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "hoarder",
+    "name": "Hoarder",
+    "description": "You don't feel right unless you're carrying as much as you can.  You suffer morale penalties for carrying less than maximum volume (weight is ignored).  Xanax can help control this anxiety.",
+    "points": -3,
+    "traits": [ "HOARDER" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "nomadic",
+    "name": "Nomadic",
+    "description": "You're too adventurous for your own good.  The more time you spend somewhere, the unhappier it makes you to be there.",
+    "points": -1,
+    "traits": [ "NOMAD", "OUTDOORSMAN" ],
+    "skills": [ { "level": 2, "name": "survival" }, { "level": 1, "name": "firstaid" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "pyromaniac",
+    "name": "Pyromaniac",
+    "description": "You have an unhealthy obsession with fire, and you get anxious if you don't light them every now and then or stand near them often.  However, you gain a mood bonus from doing so.",
+    "points": -2,
+    "skills": [ { "level": 1, "name": "cooking" } ],
+    "traits": [ "PYROMANIA" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "rigid_table_manners",
+    "name": "Rigid Table Manners",
+    "description": "You've been taught proper table manners from your early childhood on; now, you can't even think about eating without being seated at a table and taking your time.  Eating without a table and chair frustrates you, but eating like a civilized person gives you a bigger morale bonus.",
+    "points": -1,
+    "traits": [ "TABLEMANNERS" ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "truthteller",
+    "name": "Truth Teller",
+    "description": "When you try to tell a lie, you blush, stammer, and get all shifty-eyed.  Telling lies and otherwise bluffing will be much more difficult for you.",
```

Comment:
```suggestion
    "description": "When you try to tell a lie, you blush, stammer, and get all shifty-eyed.  Telling lies and otherwise bluffing will be much more difficult for you.  However, this does make you a good, honest speaker when you're not trying to hide something.",
```